CodeQualitySecurity.md

```diff
@@ -0,0 +1,13 @@
+# Code Quality and Security
+
+Egeria has a clear focus on ensuring the code delivered is of the highest quality to ensure that downstream consumers can leverage Egeria with as minimal risk as possible.
+
+Specific items that Egeria employs include but are not limited to...
+
+- Participation in the [Core Infrastructure Inititave Best Practices Badge Program](https://bestpractices.coreinfrastructure.org/en/projects/3044). This is a [free program](https://www.coreinfrastructure.org/programs/best-practices-program/) designed with the open source community with criteria that evolves to allow for compensating controls rather than a strict mechanical process.
+- [SonarCloud](https://sonarcloud.io/dashboard?id=odpi_egeria) to continously track code quality, including bugs, code smells, and potential security issues.
+- <a href="https://www.yourkit.com/java/profiler/">YourKit Java Profiler</a> for doing code profiling of Java components. YourKit supports open source projects with innovative and intelligent tools for monitoring and profiling Java and .NET.
+- [Community Bridge Vulnerability Detection](https://security.communitybridge.org) for proactive assessment of vulnerabilities in dependent libraries and packages.
+- [GitHub Security Advisories](https://help.github.com/en/github/managing-security-vulnerabilities/about-github-security-advisories) for privately discussing and managing security issues that come through (egeria-security@odpi.org) and other confidential channels.
```

Comment:
Would suggest changing this to:

```
- [GitHub Security Advisories](https://help.github.com/en/github/managing-security-vulnerabilities/about-github-security-advisories) for internally triaging security issues that come through (egeria-security@odpi.org) and other confidential channels before publishing them for broader community awareness.
```

(The intent of the security advisories is really as much about making the broader community aware as privately handling the discussions.)